src/main/java/org/nrnb/gsoc/enrichment/ui/EnrichmentTableModel.java

```diff
@@ -237,6 +237,22 @@ public void filter(List<TermSource> sources) {
         fireTableDataChanged();
     }
 
+    public void filterByEvidenceCode(Set<String> evidenceCodes) {
+        List<CyRow> rows = cyTable.getAllRows();
+        Long[] rowArray = new Long[rows.size()];
+        int i = 0;
+        for (CyRow row : rows) {
+            // implement this again
+            String termSource = row.get(EnrichmentTerm.colTermID, String.class);
+            if (evidenceCodes.size() == 0 || evidenceCodes.contains(termSource)) {
+                rowArray[i] = row.get(EnrichmentTerm.colTermID, Long.class);
```

Comment:
`rowArray[i++]` instead of writing `i++` in a separate line would work too.

Reply:
Done